### PR TITLE
Initialize sig-networking

### PR DESCRIPTION
**Pull request by Poorunga**

### CODE REVIEW

File: sig-networking/charter.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+This charter adheres to the conventions described in [KubeEdge Open Governance](https://github.com/kubeedge/community/blob/master/GOVERNANCE.md) and uses the Roles and Organization Management outlined in the governance doc.
+
+## Scope
+
+The SIG Networking focuses on research topics related to network connectivity, service collaboration, and traffic governance in the edge computing scenarios.
+
+### In scope
+#### Areas of Focus
+1. According to the network characteristics in edge computing scenarios, divide the scope into multiple layers and focus on each layer to develop solutions in depth:
+    - Edge Physical Network Layer: Physical networks in edge scenarios are generally based on area isolation, and the physical network layer is almost all hardware infrastructure (such as routers, switches, etc.). It is difficult for us to propose solutions at this layer, so we mainly focus on the above layers.
```
Comment (benjaminhuo) [inline sig-networking/charter.md:12]:
```suggestion
    - Edge Physical Network Layer: Physical networks in edge scenarios are generally based on area isolation, and the physical network layer is almost all hardware infrastructure (such as routers, switches, etc.). It is difficult for us to propose solutions at this layer, so we mainly focus on the upper layers.
```

File: sig-networking/charter.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    - Edge Tunnel Network Layer: The core task of this layer is to connect the split physical network at the lower layer, so as to shield the complexity of the physical network topology. The main capability of the tunnel network layer is to provide tunnel technology, encryption technology, etc. The main industry implementations include libp2p, ipsec, and wireguard, etc.
```
Comment (benjaminhuo) [inline sig-networking/charter.md:13]:
```suggestion
    - Edge Tunnel Network Layer: The core task of this layer is to connect the split physical network at the lower layer, so as to shield the complexity of the physical network topology. The main capability of the tunnel network layer is to provide tunnel technology, encryption technology, etc. The mainstream industry implementations include libp2p, ipsec, and wireguard, etc.
```

File: sig-networking/charter.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    - Edge Container Network Layer: The container network mainly refers to the CNI technology, but the CNI does not support data forwarding across subnets, because the CNI itself still needs the physical network to be able to communicate. Therefore, the edge container network has to rely on the capabilities of the edge tunnel network layer in order to exert the capabilities of the original cni container network configuration, packet encapsulation and routing, and networking policies. The well-known implementations in the industry include flannel, calico, and so on. This upper layer relies on the services provided by the lower layer, and this architecture is very similar to the concept of a computer network stack.
+    - Edge Virtual Network Layer: K8s provides the concept of service, which can access a service through a virtual IP address (also called cluster IP). This is the core function of the edge virtual network layer, which is mainly responsible for the transparent proxy and load balancer of the application exposed service. , the more common implementations are kube-proxy, and cilium, etc.
```
Comment (benjaminhuo) [inline sig-networking/charter.md:15]:
```suggestion
    - Edge Virtual Network Layer: K8s provides the concept of service, which can access a service through a virtual IP address (also called cluster IP). This is the core function of the edge virtual network layer, which is mainly responsible for the transparent proxying and load balancing of the application exposed service. Popular implementations include kube-proxy, and cilium, etc.
```

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: sig-networking/charter.md (hunk 1 above)
Comment (Poorunga) [reply sig-networking/charter.md]: done

File: sig-networking/charter.md (hunk 2 above)
Comment (Poorunga) [reply sig-networking/charter.md]: done

File: sig-networking/charter.md (hunk 3 above)
Comment (Poorunga) [reply sig-networking/charter.md]: done

File: sig-networking/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+The Chairs of the SIG run operations and processes governing the SIG.
+
+*TODO*
+
+### Technical Leads
+
+The Technical Leads of the SIG establish new subprojects, decommission existing subprojects, and resolve cross-subproject technical issues and decisions.
+
+- Kevin Wang (@kevin-wangzefeng), Huawei
+- Fisher Xu (@fisherxu), Huawei
```
[the remark this answers is not in the document]
Comment (Poorunga) [reply sig-networking/README.md]: done

**Comment by Poorunga** [comment]
The refresh is complete, please review.

/cc  @benjaminhuo @dingyin @fisherxu @huanwei @kevin-wangzefeng

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Comment by kevin-wangzefeng** [comment]
/lgtm
/approve

Thanks

**Pull request merged**